File: java/lib/src/main/java/com/svix/IntegrationListOptions.java
package com.svix;

import com.svix.models.Ordering;

public class IntegrationListOptions extends ListOptions {
    private Ordering order;

    public void setOrder(final Ordering order) {
        this.order = order;
    }

    public Ordering getOrder() {
        return this.order;
    }
}

File: kotlin/lib/src/main/kotlin/IntegrationListOptions.kt
package com.svix.kotlin

import com.svix.kotlin.models.Ordering

class IntegrationListOptions : ListOptions() {
    var order: Ordering? = null

    override fun iterator(iterator: String) = apply { super.iterator(iterator) }

    override fun limit(limit: Int) = apply { super.limit(limit) }

    fun order(order: Ordering) = apply { this.order = order }
}
